packages/replay/src/replay.ts

```diff
@@ -772,7 +772,7 @@ export class ReplayContainer implements ReplayContainerInterface {
     await this._addPerformanceEntries();
 
     // Check eventBuffer again, as it could have been stopped in the meanwhile
-    if (!this.eventBuffer || !this.eventBuffer.pendingLength) {
+    if (!this.eventBuffer || !this.eventBuffer.hasEvents) {
```

Comment:
I _believe_ this was actually not really correct before? as `pendingLength` could be 0 in a compression worker if everything was correctly compressed?

Reply:
`pendingLength` in the compression worker was subtracted when an event was successfully compressed (=sent to the worker). So if there would have been 10 events in the worker, but nothing pending to be sent, this would have been 0.

Reply:
Ah it doesn't matter now, but pendingLength was never subtracted, it only got reset when we finish the compression payload, so I think these two are the same

Reply:
Ah you're right of course, I misunderstood this. I guess it's fine anyhow 👍 sorry about the confusion!